WordPress/src/main/java/org/wordpress/android/ui/compose/components/shimmer/Shimmer.kt

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.ui.compose.components.shimmer
+
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.unit.IntSize
+import org.wordpress.android.ui.compose.theme.AppColor
+
+object Shimmer {
+    val color = AppColor.Black.copy(
+        alpha = 0.08F
+    )
+}
+
+@Suppress("MagicNumber")
```

Comment:
Not sure it's worth the effort, but we can introduce 2 new constants and avoid the MagicNumber annotation:
```
const val SHIMMER_START_MULTIPLIER = -2
const val SHIMMER_END_MULTIPLIER = 2
```